### ISSUE TITLE
Link color contrast ratio is too low

### ISSUE THREAD

**Issue by lucascumsille** [mysociety/alaveteli#6954]
Currently the contrast-ratio for the link on Alaveteli is 3.72, this might cause accessibility issues and also makes the links look to subtle against the white background.

Let me know @garethrees if this is an issue you'd like me to work on.

<img width="737" alt="Screenshot 2022-04-11 at 07 53 00" src="https://user-images.githubusercontent.com/13790153/162681181-e1ae53ce-16d5-4a87-8412-19b6eebfc321.png">

**Comment by garethrees** [comment]
I don't think this is a major issue on the whitelabel theme, since that's intended to be customised. Might be worth scheduling a review of whatdotheyknow-theme at some point in the future though.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
